lib/tasks/scrape.rake

```diff
@@ -51,58 +124,15 @@ namespace :scrape do
       puts "--------------------"
       puts "IN PAGE " + url
 
-      puts "PAGE TITLE "+ html_doc.css('title').to_s
-
-      data_orig_link = "a.paddedPinLink"
-      orig_link = html_doc.css(data_orig_link)
-      # puts orig_link.inspect
-      puts "SOURCE URL "+ orig_link[0]['href']
-
-      data_picture_url = "img.pinImg"
-      picture_url = html_doc.css(data_picture_url)
-      puts "PINTEREST URL " +picture_url[0]['src']
-
-      # <meta name="pinterestapp:repins" content="3776">
-      data_repin_count = "meta[name=\"pinterestapp:repins\"]"
-      repin_count = html_doc.css(data_repin_count)
-      #puts repin_count
-      if !repin_count.any?
-        puts "no repins yet"
-      else 
-        # puts repin_count
-        puts "REPINS " + repin_count[0]['content'] 
-      end
+      data_category_list = "div > div > div > a > div.name"
+      category_list = html_doc.css(data_category_list)
 
-      # <meta name="pinterestapp:likes" content="875">
-      data_like_count = "meta[name=\"pinterestapp:likes\"]"
-      like_count = html_doc.css(data_like_count)
-      #puts like_count
-      if !like_count.any?
-        puts "no likes yet"
-      else 
-        # puts like_count
-        puts "LIKES " + like_count[0]['content']
+      category_list.each do |item|
+        Category.create(name: item.text)
+        puts Category.last
```

Comment:
Not the best way to get the newly created item since you can't be sure how the database will sort the data. Instead, you should assign a variable and just refer to it.